### PR TITLE
HW3

### PR DESCRIPTION
**Pull request by geihar**
homework number 3

### CODE REVIEW

File: pavlenko_dmitryi/03/Task31.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    :type a: list, int
+    :type b: list, int
+    :return: list items that are not duplicated
+    :rtype: list
+
+    """
+
+    for i in a:
+
+            for j in b:
```
Comment (medviediev) [inline pavlenko_dmitryi/03/Task31.py]: why two indentation levels?

File: pavlenko_dmitryi/03/Task31.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+                if  j == i:
+
+                    new_list.remove(i)
```
Comment (medviediev) [inline pavlenko_dmitryi/03/Task31.py]: where is new_list declaration?

File: pavlenko_dmitryi/03/Task31.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                    new_list.remove(i)
```
Comment (medviediev) [inline pavlenko_dmitryi/03/Task31.py]: why removing i two times?

File: pavlenko_dmitryi/03/Task31.py (hunk 2 above)
Comment (geihar) [reply pavlenko_dmitryi/03/Task31.py]: new_list = list_a + list_b (25 
line)

File: pavlenko_dmitryi/03/Task31.py (hunk 3 above)
Comment (geihar) [reply pavlenko_dmitryi/03/Task31.py]: honestly speaking, I couldn’t come up with how to immediately remove 2 elements from the list

**Pull request merged**